app/javascript/components/visualization/controls/SubsampleSelector.js

```diff
@@ -16,7 +16,8 @@ function getSubsampleOptions(annotationList, clusterName) {
       clusterSubsamples = []
     }
     subsampleOptions = subsampleOptions.concat(clusterSubsamples.map(num => {
-      return { label: `${num}`, value: num }
+      // convert everything to strings to make the comparisons easier
+      return { label: `${num}`, value: `${num}` }
```

Comment:
Since "All" isn't a number, consider using `val` (or something that doesn't entail everything's a number) instead of `num`.